### ISSUE TITLE
Order Record Labels by Probability [FEATURE]

### ISSUE THREAD

**Issue by ElliotChristophers** [recognai/rubrix#4666]
**Is your feature request related to a problem? Please describe.**

Related to #4647. For a given MultilabelQuestion record, I would like to sort the labels by their probabilities. Despite, to my understanding following the instructions in #4647, sorting the probabilities for each record before sending the record to the dataset, the sorting does not work. 

My code is as follows:

`dataset = rg.FeedbackDataset.for_text_classification(
    labels=topics,
    multi_label=True,
    use_markdown=True,
    guidelines=None
)

content_class_question = rg.MultiLabelQuestion(
    name="content_class",
    title="Does the sentence include any of the following topics?",
    description="Select all that apply",
    labels=topics,
    required=True,
    visible_labels=10,
)

dataset.questions.extend([content_class_question])

records = []
for i, s in enumerate(data):
    print(i)
    prediction = []
    for topic in topics:
        prediction.append((topic, random.uniform(0,0.3) if topic not in s['topics'] else random.uniform(0.7,1)))
    q = rg.TextClassificationRecord(
        text=s['sentence_text'],
        prediction=sorted(prediction, key=lambda index : index[1],reverse=True),
        annotation=[],
        multi_label=True,
        status='Default'
    )
    records.append(q)

dataset_name ='tl1'
rg.log(records=records, workspace=workspace_name, name=dataset_name)`

So I sort the labels, and as you can see from the pictures attached, despite, when inspecting the record info the labels being sorted, they are not in the actual layout.

**Describe the solution you'd like**
I would like to be able to sort the labels by probability, for each individual record, in a straight-forward manner. I cannot imagine that this would be particularly difficult to implement.

**Describe alternatives you've considered**
I have tried logging records en bulk as well as individually, but nothing doing. Also looked at the sort button in the UI, but clearly this is for sorting records not record-labels. Same goes for metadata.

**Additional context**
![image](https://github.com/argilla-io/argilla/assets/86291929/6fba3f8e-a97c-4386-97d7-43e32aed5c6e)
![image](https://github.com/argilla-io/argilla/assets/86291929/3819bff9-116b-4310-85a5-693320016fb5)

```[tasklist]
### Tasks
```

**Comment by sdiazlor** [comment]
Hi @ElliotChristophers, ordering the labels by confidence score is not an available feature for the FeedbackDataset yet. However, it will come with the next release :). 

In addition, I've seen that you're using a FeedbackDataset but trying to add the records with `rg.log` (which is a method only used for the other datasets), instead you should use:

```python
dataset.add_records(records)
ds = dataset.push_to_argilla(name=dataset_name, workspace=workspace_name)
```

If you have any questions about the FeedbackDataset, feel free to ask here or in [Slack](https://join.slack.com/t/rubrixworkspace/shared_invite/zt-whigkyjn-a3IUJLD7gDbTZ0rKlvcJ5g) or consult the [documentation](https://docs.argilla.io/en/latest/practical_guides/create_update_dataset/records.html).

**Comment by sdiazlor** [comment]
This feature has just been released with the `1.28.0 ` version of Argilla. So, I'll close this issue.

**Comment by sdiazlor** [comment]
This feature has just been released with the `1.28.0 ` version of Argilla. So, I'll close this issue.
